etna/models/tbats.py

```diff
@@ -0,0 +1,181 @@
+from typing import Iterable
+from typing import Optional
+from typing import Tuple
+
+import pandas as pd
+from tbats.abstract import ContextInterface
+from tbats.abstract import Estimator
+from tbats.bats import BATS
+from tbats.tbats import TBATS
+from tbats.tbats.Model import Model
+
+from etna.models.base import BaseAdapter
+from etna.models.base import PerSegmentPredictionIntervalModel
+
+
+class _TBATSAdapter(BaseAdapter):
+    def __init__(self, model: Estimator):
+        self.model = model
+        self._fitted_model: Optional[Model] = None
+
+    def fit(self, df: pd.DataFrame, regressors: Iterable[str]):
+        target = df["target"]
+        self._fitted_model = self.model.fit(target)
+        return self
+
+    def predict(self, df: pd.DataFrame, prediction_interval: bool, quantiles: Iterable[float]) -> pd.DataFrame:
+        if self._fitted_model is None:
+            raise ValueError("Model is not fitted! Fit the model before calling predict method!")
+        y_pred = pd.DataFrame()
+        if prediction_interval:
+            for quantile in quantiles:
+                pred, confidence_intervals = self._fitted_model.forecast(steps=df.shape[0], confidence_level=quantile)
+                y_pred["target"] = pred
+                if quantile < 1 / 2:
+                    y_pred[f"target_{quantile:.4g}"] = confidence_intervals["lower_bound"]
+                else:
+                    y_pred[f"target_{quantile:.4g}"] = confidence_intervals["upper_bound"]
+        else:
+            pred = self._fitted_model.forecast(steps=df.shape[0])
+            y_pred["target"] = pred
+        return y_pred
+
+    def get_model(self) -> Estimator:
+        return self.model
+
+
+class _TBATSPerSegmentModel(PerSegmentPredictionIntervalModel):
+    def __int__(self, model: Estimator):
+        super().__init__(base_model=model)
+
+
+class BATSPerSegmentModel(_TBATSPerSegmentModel):
+    """Class for holding segment interval BATS model."""
+
+    def __init__(
+        self,
+        use_box_cox: Optional[bool] = None,
+        box_cox_bounds: Tuple[int, int] = (0, 1),
+        use_trend: Optional[bool] = None,
+        use_damped_trend: Optional[bool] = None,
+        seasonal_periods: Optional[Iterable[int]] = None,
+        use_arma_errors: bool = True,
+        show_warnings: bool = True,
+        n_jobs: Optional[int] = None,
+        multiprocessing_start_method: str = "spawn",
+        context: Optional[ContextInterface] = None,
+    ):
+        """Create BATSPerSegmentModel with given parameters.
+        Parameters
```

Comment:
looks like you need a blank line before `Parameters` section for correct docs rendering

you can build docs locally with this guide https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md#documentation and check if it looks good